encoders/image/ImageKerasEncoder/config.yml

```diff
@@ -10,8 +10,7 @@ requests:
     [IndexRequest, SearchRequest]:
       - !EncodeDriver
         with:
-          granularity_range: [1, 1]
```

Comment:
I think Drivers should be removed by default, is this the only encoder with a Driver attached different than the default ones in the config.yml?

Reply:
Yes it is.

Reply:
So let's remove it, I think we should keep default config, any user can later adapt it, but for coherence as well I think Drivers should not be changed by default.

Reply:
No, what I meant is, that it is the only executor in the hub, that explicitly does not need the default, yet (which is `['c', 'r']`.

Reply:
I do not see why this should behave differently than ImageTorchEncoder?

Reply:
And why shouldn't it be used to encode the root?